docs/content/3.guide/1.writing/3.mdc.md

```diff
@@ -200,6 +200,37 @@ defineProps(['type'])
   ::
 ::
 
+Multiple props can be separated with a space:
+
+```md
+::alert{type="warning" icon="exclamation-circle"}
+Oops! An error occurred
+::
+```
+
+If you want to pass arrays or objects as props to components you can pass them as JSON string and prefix the prop key with a colon to automatically decode the JSON string.
+Note that in this case you should use single quotes for the value string so you can use double quotes to pass a valid JSON string:
+
+::code-group
+
+  ```md [array.md]
+  ::dropdown{:items='["Nuxt", "Vue", "React"]'}
+  ::
+  ```
+
+  ```md [number-array.md]
+  ::dropdown{:items=[1,2,3.5]}
```

Comment:
Could you please wrap this one with double quotes too? This is not well tested feature and may create some issues for users.
Let's keep suggesting valid JSON string until we create a proper solution for non string values